src/atc_api/main.go

```diff
@@ -70,6 +71,7 @@ func ParseArgs() Arguments {
 	warn_only := flag.Bool("W", false, "only warn if the thrift server isn't reachable")
 	ipv4 := flag.String("4", "", "IPv4 address for the API")
 	ipv6 := flag.String("6", "", "IPv6 address for the API")
+	proxy_addr := flag.String("proxy-addr", "", "Address of proxy authorized to set X_HTTP_REAL_IP")
```

Comment:
We should use [kingpin](https://github.com/alecthomas/kingpin) for our arguments. It'll allow `--proxy-addr` instead of `-proxy-addr`, which is [better](http://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html).